app/models/api/openid_connect/scope.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module OpenidConnect
+    class Scope < ActiveRecord::Base
+      has_many :authorizations, through: :authorization_scopes
+
+      validates :name, presence: true, uniqueness: true
+    end
+  end
+end
```

Comment:
I see some regarding having a serialized column, any reason this isn't dropped yet?